diff --git a/include/linux/cgroup_subsys.h b/include/linux/cgroup_subsys.h
index 445235487230..a3ed3464d361 100644
--- a/include/linux/cgroup_subsys.h
+++ b/include/linux/cgroup_subsys.h
@@ -45,18 +45,14 @@ SUBSYS(net_cls)
 SUBSYS(perf_event)
 #endif
 
-#if IS_ENABLED(CONFIG_CGROUP_NET_PRIO)
-SUBSYS(net_prio)
+#if IS_ENABLED(CONFIG_CGROUP_PIDS)
+SUBSYS(pids)
 #endif
 
 #if IS_ENABLED(CONFIG_CGROUP_HUGETLB)
 SUBSYS(hugetlb)
 #endif
 
-#if IS_ENABLED(CONFIG_CGROUP_PIDS)
-SUBSYS(pids)
-#endif
-
 #if IS_ENABLED(CONFIG_CGROUP_RDMA)
 SUBSYS(rdma)
 #endif
